src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier(random);
+    private final int limit = 101;
```

Comment:
variable may be constant
```suggestion
public class Lottery {
    private static final int NUMBER_LIMIT = 101;

    private final Random random = new Random();
    private final ColorSupplier colorSupplier = new ColorSupplier(random);
```

Reply:
Checkstyle plugin gives an error on the constant name with more than two capital letters in the row
https://prnt.sc/MXd6QETFQo4I